staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go

```diff
@@ -207,8 +207,6 @@ func (w *WebhookAuthorizer) Authorize(ctx context.Context, attr authorizer.Attri
 		}
 	}
 	switch {
-	case r.Status.Denied && r.Status.Allowed:
```

Comment:
This is guarding against bugs in TokenReview implementors. pkg/registry isn't the only implementer